src/pages/SpeciesCompendia/index.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import Button from '../../components/Button';
+import './SpeciesCompendia.scss';
+import EmailSection from '../Main/EmailSection';
+import PointsBackground from './PointsBackground';
+import DownloadCompendia from './DownloadCompendia';
+
+export default function SpeciesCompendia() {
+  return (
+    <div>
+      <Helmet>
+        <title>Species Compendia</title>
+      </Helmet>
+
+      <div className="species-compendia">
+        <div className="species-compendia__header">
+          <PointsBackground />
+
+          <div className="species-compendia__header-inner">
+            <h1 className="species-compendia__title">
+              Species Compendia are the collection of all the samples available
+              on refine.bio, aggregated and normalized by species.{' '}
+            </h1>
+
+            <DownloadCompendia />
+          </div>
+        </div>
+
+        <div className="layout__content">
+          <div className="species-compendia__section">
+            <div className="species-compendia__section-title">
+              Data scientists and computational biologists can use species
+              compendia to quickly construct training and testing sets and
+              extract features from a diverse set of biological contexts and
+              conditions
```

Comment:
Do we need a period at the end of this sentence?

Reply:
yes
```suggestion
              conditions.
```